packages/example-ui/src/components/common/Checkout/CreatePaymentForm.tsx

```diff
@@ -125,7 +125,7 @@ const CreatePaymentForm = (
                     }}
                     error={!!errors?.sellerAccountId}
                   >
-                    <MenuItem data-safe-name='Pleasant View Gardens' value={"acc_Yzem4ZF4mKnMO0dBObRUU"}>
+                    <MenuItem data-safe-name='Pleasant View Gardens' value={"acc_37klepovVgsdN5bkWSP1I4"}>
```

Comment:
Would be amazing if this could use the seller picker instead of this other random list. Follow up?